lib/grant.js

```diff
@@ -498,6 +499,24 @@ function saveRefreshToken (done) {
   });
 }
 
+/**
+ * Check if MFA is enabled for the user. If MFA is enabled save an mfa token
+ * and return an error with the mfa token
+ *
+ * @param  {Function} done
+ * @this   OAuth
+ */
+function checkMfa (done) {
+  if (this.user && this.user.mfaEnabled) {
```

Comment:
is this a particle user object? where is `mfaEnabled` coming from?

Reply:
This is a Particle User object.
https://github.com/particle-iot/api-service/pull/503/files#diff-764e925abfdd0323a27925ec2bda30e1R335